@xen-orchestra/web/src/components/hosts/hostsTable.vue

```diff
@@ -0,0 +1,231 @@
+<template>
+  <div class="host-table">
+    <div class="table-actions">
+      <UiQuerySearchBar @search="value => (searchQuery = value)" />
+      <UiTopBottomTable :selected-items="0" :total-items="0">
+        <UiTablePagination v-bind="paginationBindings" />
+      </UiTopBottomTable>
+    </div>
+    <VtsDataTable
+      :no-data-message="hosts.length === 0 ? t('no-hosts-available') : undefined"
+      :has-error
+      :is-ready="arePifsReady && hostReady"
+    >
+      <template #thead>
+        <tr>
+          <template v-for="column of visibleColumns" :key="column.id">
+            <th v-if="column.id === 'checkbox'" class="checkbox">
+              <div v-tooltip="t('coming-soon')">
+                <UiCheckbox disabled :v-model="areAllSelected" accent="brand" @update:model-value="toggleSelect" />
+              </div>
+            </th>
+            <th v-else-if="column.id === 'more'" class="more">
+              <UiButtonIcon v-tooltip="t('coming-soon')" icon="fa:ellipsis" accent="brand" disabled size="small" />
+            </th>
+            <th v-else>
+              <div v-tooltip class="text-ellipsis">
+                <VtsIcon accent="brand" size="medium" :name="headerIcon[column.id]" />
+                {{ column.label }}
+              </div>
+            </th>
+          </template>
+        </tr>
+      </template>
+      <template #tbody>
+        <tr
+          v-for="row of spacesRecords"
+          :key="row.id"
+          :class="{ selected: selectedHostId === row.id }"
+          @click="selectedHostId = row.id"
+        >
+          <td
+            v-for="column of row.visibleColumns"
+            :key="column.id"
+            class="typo-body-regular-small"
+            :class="{ checkbox: column.id === 'checkbox' }"
+          >
+            <div v-if="column.id === 'checkbox'" v-tooltip="t('coming-soon')">
+              <UiCheckbox v-model="selected" disabled accent="brand" :value="row.id" />
+            </div>
+            <UiButtonIcon
+              v-else-if="column.id === 'more'"
+              v-tooltip="t('coming-soon')"
+              icon="fa:ellipsis"
+              accent="brand"
+              disabled
+              size="small"
+            />
+            <div v-else-if="column.id === 'host'">
+              <UiObjectLink :route="`/host/${column.value.id}/`">
+                <template #icon>
+                  <VtsObjectIcon size="medium" :state="isHostRunning(column.value.state)" type="host" />
+                </template>
+                {{ column.value.label }}
```

Comment:
```suggestion
      <span v-tooltip="column.value.label">{{ column.value.label }}</span>
```
<img width="437" height="507" alt="image" src="https://github.com/user-attachments/assets/49aea175-c47e-45a4-a52d-1e962341cf9d" />

Reply:
I think we don't need to add this here 
Because in the `UiObjectLink` component we already have v-tooltip